node/src/sign.rs

```diff
@@ -45,7 +45,17 @@ pub async fn pre_sign(
             threshold,
         },
     )?;
+    let start = SystemTime::now();
+
     let presignature = run_protocol("presign", channel, me, protocol).await?;
+
+    let duration = start.elapsed()?.as_millis();
+    metrics::MPC_PRE_SIGNATURE_TIME_MS
```

Comment:
I would suggest to convert these metrics to buckets instead:
 * A bucket metric would automatically keep track of the count and sum, so you don't need to have instance or cumulative.
 * A bucket metric would store a histogram so we can see what are the percentile stats.
 * The calling code doesn't need to do it twice, and doesn't need to incur the overhead of looking up by string keys.

Reply:
Sorry I meant Histogram, not bucket.